plasma_framework/contracts/poc/fast_exits/Liquidity.sol

```diff
@@ -0,0 +1,186 @@
+            ownerFirstTxOutput == msg.sender,
+            "Was not called by the first Tx owner"
+        );
+    }
+
+    /**
+     * @notice Verify the First Tx provided is valid
+     * @param utxoPosInput position of the output that created the inputs for second transaction
+     * @param rlpInputCreationTx RLP-encoded first transaction that transfers to this contract
+     * @param inputCreationTxInclusionProof First transactions inclusion proofs
+    */
+    function verifyTxValidity(
+        uint256 utxoPosInput,
+        bytes memory rlpInputCreationTx,
+        bytes memory inputCreationTxInclusionProof
+    ) internal {
+        PosLib.Position memory utxoDecoded = PosLib.decode(utxoPosInput);
+        utxoDecoded.outputIndex = 0;
+        (bytes32 root, ) = plasmaFrameworkInstance.blocks(utxoDecoded.blockNum);
+        require(root != bytes32(""), "Failed to get root of the block");
+        bool txExists = Merkle.checkMembership(
+            rlpInputCreationTx,
+            utxoDecoded.txIndex,
+            root,
+            inputCreationTxInclusionProof
+        );
+        require(
+            txExists,
+            "Provided Transaction isn't finalized or doesn't exist"
+        );
+    }
+
+    /**
+     * @notice func that calls omg-contracts to start the exit
+     * @param utxoPosToExit position of the output which the contract has to exit
+     * @param rlpOutputTxToContract RLP-encoded transaction that creates the output for the contract
+     * @param outputTxToContractInclusionProof Second Transaction's inclusion proof
+    */
+    function runExit(
+        uint256 utxoPosToExit,
+        bytes memory rlpOutputTxToContract,
+        bytes memory outputTxToContractInclusionProof
+    ) private {
+        PaymentStandardExitRouterArgs.StartStandardExitArgs memory s;
+        s.utxoPos = utxoPosToExit;
+        s.rlpOutputTx = rlpOutputTxToContract;
+        s.outputTxInclusionProof = outputTxToContractInclusionProof;
+        paymentExitGame.startStandardExit.value(msg.value)(s);
+    }
+
+    function getContractBalance() public view returns(uint256) {
+        return address(this).balance;
+    }
+
+    /**
+     * @dev Get Amount from contract after exit is processed - (to be updated)
+     * @param exitId The exit id
+    */
+    function getWithdrawal(uint160 exitId) public {
+        require(
+            exitIdtoUser[exitId] == msg.sender,
+            "Only the exitId owner can get the withdrawal"
+        );
+        uint160[] memory exitIdList = new uint160[](1);
+        exitIdList[0] = exitId;
+        PaymentExitDataModel.StandardExit[] memory exits = paymentExitGame.standardExits(
+            exitIdList
+        );
+        if (exits[0].utxoPos == 0) {
+            exitIdtoUser[exitId] = 0x0000000000000000000000000000000000000000;
+            msg.sender.transfer(exitIdtoAmount[exitId]);
+        } else {
+            revert("Not processed exit");
+        }
+    }
+
+    function() external payable {
```

Comment:
minor: can we have a comment/document on this function and state that we need the fallback function to allow transfer fund to this smart contract. (This is the reason of this empty function here right?)

Reply:
done